.changelog/23319.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:security
+core: Adding escape to arbitrary file access on streamAllocDir
```

Comment:
The changelog should be user facing, whereas this is more describing the internals we changed. We should instead describe what security flaw we fixed. Or, in cases where we're just silencing false positive alerts with no user-visible behavior, we should omit the changelog.

So something like this:

```suggestion
migration: Added a check for relative paths escaping the allocation directory when unpacking archive during migration, to harden clients against compromised peer clients sending malicious archives
```

(If I understand the issue we're resolving here.)